sys/kern/subr_intr.c

```diff
@@ -1867,8 +1864,6 @@ intr_ipi_setup(u_int ipi, const char *name, intr_ipi_handler_t *hand,
 	if (error != 0)
 		return;
 
-	isrc->isrc_handlers++;
-
```

Comment:
This turns out to be an issue.  With `intr_setup_irq()` the handler can be detected via `CK_SLIST_EMPTY()`, but here the IPI interrupt handler is never passed to the event core.  This makes sense due to the differences between IPI and SPI interrupts, but complicates the detection.

Would be possible to synthesize a NOP handler function to hand to the event core.  Nominally that _should_ never be called and isn't much more than a placeholder for this purpose.  Likely a worthwhile trade-off as this shrinks every `struct intr_irqsrc` by 4 bytes by creating an extra `struct intr_handler` for IPI interrupts (roughly +64 bytes).

Due to this issue I'm pulling this commit from the series for now.